week3_homework(continue-week2)/contact.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div id="blog_logo">My Blog</div>
+        <nav>
+            <a href="home.html"><button>Home</button></a>
+            <a href="contact.html"><button>Contact</button></a>
+        </nav>
+    </header>
+    <main>
+        <div style="display: block;">
+            <label for="name">Nume și prenume: </label>
+            <input type="text">
+            <br>
+            <label for="email">Email-ul tău</label>
+            <input type="text">
+            <br>
+            <p class="space-top">Adauga aici comentariul tau</p>
+            <textarea placeholder="Adaugă aici comentariul tău." rows="4" cols="60">
+            </textarea> <br>
+            <button type="submit" style="margin-top: 10px;">Submit</button>
+        </div>
+
+        <button onclick="topFunction()" id="myBtn" title="Go to top">Top</button>
+        <script>
+            //Get the button
+            var mybutton = document.getElementById("myBtn");
+
+            // When the user scrolls down 20px from the top of the document, show the button
+            window.onscroll = function () { scrollFunction() };
```

Comment:
in loc sa duplici codul, se poate pune intr-un fisier de javascript si importa peste tot